app/actions/GroupActions.js

```diff
@@ -278,5 +278,5 @@ export function removeGroup(id: string, group: Object): Thunk<*> {
               : 'Sletting av gruppe fullført',
         },
       })
-    ).then(() => dispatch(push('/interestgroups/')));
+    ).then(() => dispatch(push('/interestGroups/')));
```

Comment:
Why are these camelCased?

Reply:
well, I just thought it made more sense, since in English the term "interessegruppe" is split into two words 😅 There was some camelCase elsewhere, so I thought I might as well make it consistent. I can switch back if you hate it though

Reply:
Okay I see.
Yes, I hate it :grimacing:. All urls should be lowercase IMO. I think you should have made it consistent the other way and made the camelcase urls lowercase.

Reply:
Sure thing, I'll switch it up

Reply:
Since we have very few of these I don't really "care". In practice, kebab case is the correct thing to do.

Reply:
So, what do we want here? lowercase or kebab-case?

Reply:
If you're first going to change it, I want kebab

Reply:
Nope. Whatever makes the most sense is good!